app/Resources/views/admin/adherent_status.html.twig

```diff
@@ -5,11 +5,15 @@
         {% if object.isReferent %}
             <span class="label label-primary" style="padding: 2px;">Référent</span><br/>
         {% endif %}
+
+        {% if object.isLegislativeCandidate %}
```

Comment:
You could remove all prefix
`object.legislativeCandidate` == `object.isLegislativeCandidate`